core/src/main/java/org/apache/accumulo/core/util/NanoTime.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.core.util;
+
+import java.time.Duration;
+
+/**
+ * This class implements a strong type for System.nanoTime() that offers the limited operations that
+ * can be performed on a nanoTime.
+ */
+public final class NanoTime {
+  private final long nanoTime;
```

Comment:
Do you think it would improve readability if this was named nanos (or timestamp, value, or something) to distinguish that it is the nanoseconds measurement, not an instance of this class?

Reply:
Looked at the javadoc for nanoTime and derived a name from that.